packages/e2e-tests/test/e2e-aws.spec.ts

```diff
@@ -295,4 +298,86 @@ describe('e2e AWS AUTH', function () {
       });
     });
   });
+
+  context('with aws profile', function () {
+    const awsConfigPath = path.join(
+      os.tmpdir(),
+      `aws-config-${new Date().getTime()}-${Math.random()}`
+    );
```

Comment:
`useTmpdir()` from `./repl-helpers.ts`?

Reply:
Didn't know about that. copy/pasted this from another test.